spacy/tests/pipeline/test_spancat.py

```diff
@@ -397,3 +397,21 @@ def zero_suggester(docs, *, ops=None):
     assert set(spancat.labels) == {"LOC", "PERSON"}
 
     nlp.update(train_examples, sgd=optimizer)
+
+
+def test_suggest_candidates():
+    nlp = Language()
+    spancat = nlp.add_pipe("spancat", config={"spans_key": SPAN_KEY})
+    train_examples = make_examples(nlp)
+    nlp.initialize(get_examples=lambda: train_examples)
+    texts = [
+        "Just a sentence.",
+        "I like London and Berlin",
+        "I like Berlin",
+        "I eat ham.",
+    ]
+    docs = [nlp(text) for text in texts]
+    output = spancat.suggest_candidates(docs)
+
+    assert len(output) == len(texts)
+    assert type(output[0].spans["candidates"]) == SpanGroup
```

Comment:
You could probably also `assert` the actual contents here, using the default suggester?